lib/main.dart

```diff
@@ -1,4 +1,9 @@
 import 'package:flutter/material.dart';
+import 'package:movie_card/widgets/like_button.dart';
+import 'package:movie_card/widgets/row_genres.dart';
+import 'package:movie_card/widgets/row_stars.dart';
+
+import 'utils/constants.dart';
```

Comment:
For the future: choose whether using relative or full paths for imports and stick to it, to keep consistency in your code!

Reply:
Perfect, i will keep it in mind! Thank you